tests/modules/subcommands.c

```diff
@@ -42,23 +42,27 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
         return REDISMODULE_ERR;
     RedisModuleCommand *subcmd = RedisModule_GetCommand(ctx,"subcommands.bitarray|set");
 
+#ifdef INCLUDE_UNRELEASED_KEYSPEC_API
```

Comment:
bty, in line 32, `int spec_id`, there is a warning. but it is not a problem
```
subcommands.c:32:9: warning: unused variable ‘spec_id’ [-Wunused-variable]
```

Reply:
let's solve it for the RC, can you make a PR (i can't merge mine)

Reply:
sure, #10184

Reply:
I'll handle it

Reply:
Here's a PR: #10185

Reply:
Ah sorry! Same time, identical fix. :)